playbooks/templates/suricata_overrides.yaml.j2

```diff
@@ -30,7 +30,8 @@ outputs:
   - eve-log:
       enabled: yes
       filetype: regular
-      filename: eve.json
+      filename: eve-%Y-%m-%d-%H.json
```

Comment:
I removed it for now. That's not supported until the latest release candidate; I just forgot to take it back out